README.md

```diff
@@ -62,6 +62,7 @@ The Ginkgo DPC++ module has the following __additional__ requirements:
 
 * _OneAPI 2021.3+_
 * Set `dpcpp` as the `CMAKE_CXX_COMPILER`
+* `C++17` is used to compile this module, while the rest of Ginkgo is compiled using `c++14`.
```

Comment:
nit: I think this should either be `c++17` with a lower case `c`, or `C++ 17` with a space.